types/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// TypeScript Version: 2.4
+
+import "./vue"
+
+import { Vuelidate } from 'vuelidate/types/vuelidate'
```

Comment:
if you leave out the `declare module 'vuelidate/types/vuelidate'` from the `vuelidate.d.ts` then you can just do `import ... from 'vuelidate'`